configs/setups/awiesm/awiesm-2.2.yaml

```diff
@@ -71,6 +71,13 @@ echam:
                         nproca: 24
                         nprocb: 18
 
+        add_export_vars:
```

Comment:
Do this actually work out of a `<...>_environment_changes` block?

Reply:
...it does not